### PR TITLE
Improvements for Day One face

### PR DESCRIPTION
**Pull request by Kistelini**
- code cleanup
- allow years until 2080 in the future, not just the current year (already possible for month and day)
- enable quick cycling though the settings
- show set date on short alarm button press

### CODE REVIEW

File: movement/watch_faces/complication/day_one_face.c [hunk 1]
```diff
@@ -85,96 +113,143 @@ bool day_one_face_loop(movement_event_t event, movement_settings_t *settings, vo
...
+                    // fall through
+                case PAGE_MONTH:
+                    // fall through
+                case PAGE_DAY:
+                    _day_one_face_abort_quick_cycle(state);
+                    _day_one_face_increment(state);
+                    break;
+                case PAGE_DISPLAY:
+                    state->current_page = PAGE_DATE;
+                    sprintf(buf, "%04d%02d%02d", state->birth_year, state->birth_month, state->birth_day);
```
Comment (WesleyAC) [inline movement/watch_faces/complication/day_one_face.c:219]: This triggers a (somewhat spurious) warning about a possible overflow. While it won't happen, could you still change this to something like:

```c
sprintf(buf, "%04d%02d%02d", state->birth_year % 10000, state->birth_month % 100 , state->birth_day % 100);
```

to silence it?

File: movement/watch_faces/complication/day_one_face.h [hunk 2]
```diff
@@ -30,13 +30,22 @@
 // The Day One face is designed to count upwards from the wearer's date of birth. It also functions as an
```
Comment (WesleyAC) [inline movement/watch_faces/complication/day_one_face.h:30]: Could you change this to say that it counts the days since or until a given date?

**Review by WesleyAC — CHANGES_REQUESTED** [review]
Looks good overall, just two minor things

File: movement/watch_faces/complication/day_one_face.h (hunk 2 above)
Comment (Kistelini) [reply movement/watch_faces/complication/day_one_face.h]: Done!

**Pull request merged**